### ISSUE TITLE
Support custom config on nestjs-admin CLI

### ISSUE THREAD

**Issue by therealedsheenan** [Theodo-UK/nestjs-admin#148]
### Summary
It would be nice to have a support for a custom ormconfig.js when generating an admin user via CLI. Something like: `npx nestjs-admin createAdminUser --config=<CUSTOM_PATH>/config.js`.

### Current workaround
I needed to copy my custom `ormconfig.js` file to the root of the codebase and fire up the cli command from there. (`npx nestjs-admin createAdminUser `).

**Comment by ruslanchek** [comment]
Lucky you! I'm unable to start it at all, too many bugs and nothing works from the readme. Literally nothing works.

**Comment by TheBigBadBunce** [comment]
Sounds good. Added to the todo list!
@ruslanchek - Can you share some of the bugs you've encountered? It's about time we squashed a few 😅

**Comment by williamdclt** [comment]
It is indeed an important feature.

At the moment, the `nestjs-admin` script works if you use the standard conventions of TypeOrm (either top-level ormconfig.* or env vars, hardcoded config in `TypeOrm.forRoot` can't be supported). And even then, we'll always use the `default` database connection.

I am torn between several options (thinking out loud here):

1. providing CLI options to specify your config file and the database connection you want to use
    - Good: it's fairly easy to use
    - Bad: this will be a bit tricky to implement. We'll need to mimic TypeOrm's `--config` param, probably using [ConnectionOptionsReader](https://github.com/typeorm/typeorm/blob/master/src/connection/ConnectionOptionsReader.ts) which isn't documented.
    - Bad: this will probably need non-trivial maintainance 
2. Always prompting for a database URL (or maybe only if we can't detect a TypeOrm config automatically)
    - Good: will always work, easy to maintain, less dependency on the TypeOrm config loading
    - Bad: less-than-ideal UX
3. Getting rid of this `createAdminUser` altogether, adding documentation on how to roll your own to fit your particular needs
    - Good: less maintainance cost, more flexible
    - Bad: significantly harder to get started with the lib

Given that this is a script meant to be ran once per environment (to create an initial AdminUser, which can then create more through the admin interface), I'm not super happy about maintainance burden. That being said, it's one of the first interactions of the user with this library and I want it to be a good experience...

All in all, I'd probably go with the first option. I will post updates here once this is done (if anybody wants to work on this please do and announce it here)
